mergebot-config.json

```diff
@@ -0,0 +1,39 @@
+{
+    "override-status": {
+      "enabled": true,
+      "jira_label": "mergebot-override"
+    },
+    "test": true,
+    "backport": true,
+    "update_jira_on_merge": {
+      "enabled": true,
+      "interactive": false,
+      "fix_version_map": {
+        "main": "Kommander 2.2.0",
+        "release-2.1": "Kommander 2.1.1"
```

Comment:
we don't have a release branch for 2.1, should I both keeping `fix_version_map` here at all?

Reply:
hmm yeah you can remove `release-2.1` from the map. we can leave main there pointing to the next version.